department-of-veterans-affairs/va.gov-cms
# VAGOV-000 hub icon migration fix

To test:
1. Go to /admin/structure/migrate/manage/va_new_benefits/migrations/va_hub/execute and execute import.
2. Go to /section/veterans-benefits-administration and edit "VA housing assistance".
3. Edit "VA housing assistance" and confirm that Hub Icon value is "Housing".
